src/main/java/com/techsophy/tsf/account/utils/Rsa4096.java

```diff
@@ -0,0 +1,97 @@
+package com.techsophy.tsf.account.utils;
+
+import com.techsophy.tsf.account.config.GlobalMessageSource;
+import com.techsophy.tsf.account.dto.UserFormDataSchema;
+import com.techsophy.tsf.account.exception.BadRequestException;
+import com.techsophy.tsf.account.exception.RunTimeException;
+import com.techsophy.tsf.account.exception.UnAuthorizedException;
+import javax.validation.ConstraintViolationException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.*;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.KeySpec;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+
+import static com.techsophy.tsf.account.constants.ErrorConstants.UN_AUTHORIZED_EXCEPTION;
+
+public class Rsa4096 {
+    private GlobalMessageSource globalMessageSource;
+    private KeyFactory keyFactory;
+    private PublicKey publicKey;
+    public Rsa4096() throws NoSuchAlgorithmException, IOException, InvalidKeySpecException {
+        setKeyFactory();
+        setPublicKey("keys/public_key_rsa_4096_pkcs8-exported.pem");
+    }
+
+    protected void setKeyFactory() throws NoSuchAlgorithmException {
+        this.keyFactory = KeyFactory.getInstance("RSA");
+    }
+
+    protected void setPublicKey(String classpathResource)
+            throws IOException, InvalidKeySpecException {
+        InputStream is = this
+                .getClass()
+                .getClassLoader()
+                .getResourceAsStream(classpathResource);
```

Comment:
class path resource? this should not be committed to the jar

Reply:
We need the placeholder where we need to keep the Keyfile